Documentation/servicing/tagging.md

```diff
@@ -35,4 +35,11 @@ git push 'git@github.com:dotnet/source-build' "v${runtimeVersion}-runtime" && \
 git push 'git@github.com:dotnet/source-build' "v${sdkVersion}-SDK"
 ```
 
-**Do not use "git push --tags"**. Unless you have a fresh repo with no other tags, this will push all your tags. This risks messing up existing tags on the remote.
\ No newline at end of file
+If `upstream` is configured as `dotnet/source-build` command can be:
```

Comment:
```suggestion
If `upstream` is configured as `dotnet/source-build`, you can use that instead. You may need to do this, for example, if you don't use SSH authentication. If this applies to you, run this command instead of the fully-specified push:
```